sdk/communication/Azure.Communication.Messages/Configuration.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
`Communiction.Messages` set `save-inputs: true` in tspConfig.yaml (https://github.com/Azure/azure-rest-api-specs/blob/44cbf51b9eee8a2dbf55ced61a61032aaed0148b/specification/communication/Communication.Messages/tspconfig.yaml#L36C5-L36C22), so the `Configuration.json` and `tspCodeModel.json` are maintained.

Reply:
I wonder why these are new files, instead of being moved to here.
Could you investigate?
I recall this RP has a `tspCodeModel.json` file, and it gets updated everytime we change the schema of tspCodeModel file, therefore here we should expect to see the old files are removed, and created in the current location

Reply:
We updated the location of the `tspCodeModel.json` and `Configuration.json` in autorest.csharp, so it cannot be deleted by current `autorest.csharp` codegen.
We need to manually delete previous `tspCodeModel.json` and `Configuration.json` file, then it will show as rename.

I will manually do this when upgrade azure-sdk-for-net emitter version.